arangosh/Shell/ClientFeature.cpp

```diff
@@ -235,6 +235,23 @@ void ClientFeature::validateOptions(std::shared_ptr<ProgramOptions> options) {
     FATAL_ERROR_EXIT();
   }
 
+  if (_endpoint.find("0.0.0.0") != std::string::npos) {
+    LOG_TOPIC("701fb", ERR, arangodb::Logger::FIXME)
+        << "invalid value for --server.endpoint ('" << _endpoint <<
+      "') - 0.0.0.0 is only allowed for servers binding any." <<
```

Comment:
To be honest, I don't understand the error message "0.0.0.0 is only allowed for servers binding any".
Can we phrase it?